src/model/write/request.rs

```diff
@@ -0,0 +1,473 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::collections::{BTreeMap, HashMap};
+
+use ceresdbproto::storage::{
+    Field, FieldGroup as FieldGroupPb, Tag as TagPb, WriteEntry as WriteEntryPb,
+    WriteMetric as WriteMetricPb, WriteRequest as WriteRequestPb,
+};
+
+use crate::model::value::{TimestampMs, Value};
+
+#[inline]
+fn is_key_word(name: &str) -> bool {
+    name.eq_ignore_ascii_case("tsid") && name.eq_ignore_ascii_case("timestamp")
+}
+
+/// Write request
```

Comment:
These comments in this file may need some polishes. About the comment style refer to this [example](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#example-1).